trigger/task_test.go

```diff
@@ -292,6 +292,50 @@ func (s *taskSuite) SetupTest() {
 	s.t.uiConfig = *ui
 }
 
+func (s *taskSuite) TestTriggerEvent() {
+	s.NoError(db.ClearCollections(task.Collection, event.SubscriptionsCollection))
```

Comment:
shouldn't this be in setup test and/or teardown group?

Reply:
No, I'm relying on an already existing setup test, but don't want all of its setup code. This overrides part of it.